cli/root.go

```diff
@@ -687,6 +687,14 @@ func namedWorkspace(ctx context.Context, client *codersdk.Client, identifier str
 	return client.WorkspaceByOwnerAndName(ctx, owner, name, codersdk.WorkspaceOptions{})
 }
 
+func docsURL(ctx context.Context, client *codersdk.Client) string {
```

Comment:
I would rather we fetch this once on startup at the same time we do stuff like the version check and store it on `r`

Reply:
I had this initially, but nothing else on `r` is fetched. The URL is used infrequently enough that I don't think its worth the cost of doing an extra request if we don't need it. I guess we could have a middleware like `initClient` that fetches the deployment config and sets defaults?